### DISCUSSION TITLE
Save layout

### DISCUSSION THREAD

**Discussion by p3rh5g** [postmanlabs/postman-flows#404] [Q&A]
Is it possible to save the layout? Mine contantly keeps rearranging everything if I close and open the flow later.

**Comment by samay-v** [comment]
Hey @Palpie, your layout autosaves today is Flows. Is it possible that you have flows open in another tab or you're collaborating on the flow with someone else in the team?

**Comment by p3rh5g** [comment]
It is a private flow and I only had it open in one tab in the Windows application. The problem seems to have disappeared after I restarted Postman.
